core/unit_test/TestViewMemoryAccessViolation.hpp

```diff
@@ -46,6 +46,9 @@
 
 #include <gtest/gtest.h>
 
+// FIXME_SYCL
+#if !(defined(KOKKOS_COMPILER_INTEL) && defined(KOKKOS_ENABLE_SYCL))
```

Comment:
Please confirm that both `view_memory_access_violations_from_{host,device}` unit test are segfaulting